arch/arm/soc/nordic_nrf5/nrf52/soc.h

```diff
@@ -24,9 +24,4 @@
 #define FLASH_PAGE_ERASE_MAX_TIME_US 	89700UL
 #define FLASH_PAGE_MAX_CNT		256UL
 
-/* For IMG_MANAGER  */
```

Comment:
I'd rather we keep FLASH_DRIVER_NAME here, and not duplicate it in all the fixup files.  Hopefully we can get ride of it in the future.

So can we keep the comment and such and just have:

#define FLASH_DRIVER_NAME FLASH_DEV_NAME